src/Session.php

```diff
@@ -378,6 +378,30 @@ public static function initNavigateListItems($itemtype, $title = "", $url = null
     }
 
 
+    /**
+     * Check if active entities should be reloaded.
+     *
+     * @return bool true if active entities should be reloaded, false otherwise
+     */
+    public static function shouldReloadActiveEntities(): bool
+    {
+        $glpiactiveentities = $_SESSION['glpiactiveentities'] ?? [];
```

Comment:
The `inc/includes.php` file is loaded everywhere, even in the console, and in that case, I guess that some session variables are not defined at all. Adding the following check could be safer, to prevent extra queries on edge cases.

```suggestion
        if (!array_key_exists('glpiactive_entity')) {
            return false;
        }
        $glpiactiveentities = $_SESSION['glpiactiveentities'] ?? [];
```